samtranslator/model/apigateway.py

```diff
@@ -285,7 +285,7 @@ def _is_missing_identity_source(self, identity):
         try:
             ttl_int = int(ttl)
         # this will catch if ttl is None and not convertable to an int
-        except TypeError:
+        except (TypeError, ValueError):
```

Comment:
can you add a unit test case to show when exactly this exception will be thrown?
My understanding is ValueError will be thrown if ttl value contains a non numeric value, and I think in this case we should throw a different exception

Reply:
Thanks, test cases added.